src/state/sagas/buildPageSaga.js

```diff
@@ -6,15 +6,17 @@ import {
   version as schemaVersion,
 } from '../../components/url-state/schema/BuildPageSchema_v2';
 import { buffsDict } from '../../components/url-state/schema/SchemaDicts';
+import { PARAMS } from '../../utils/queryParam';
 import { changeBuildPage, changeCharacter } from '../slices/buildPage';
+import { changeGameMode } from '../slices/userSettings';
 import SagaTypes from './sagaTypes';
 
 // channels solve the problem "how to get value out of callback"
 // https://stackoverflow.com/questions/43031832/how-to-yield-put-in-redux-saga-within-a-callback
 const compressChannel = channel();
 function* exportStateCharacter({ newPage, copyToClipboard }) {
   const { optimizer } = yield select();
-  const { buildPage, control } = optimizer;
+  const { buildPage, control, userSettings } = optimizer;
```

Comment:
in spots like this you can do `const state = yield select()` and import selectors and call them on `state` if you want to decouple the code from the slice shape; I don't think it really matters but it's a noteworthy option for when the slice is really complicated